onvm/onvm_nflib/onvm_event.h

```diff
@@ -135,8 +144,12 @@ int nf_subscribed_to_event(struct event_tree_node *root, uint64_t event_id, uint
 struct event_tree_node *get_event(struct event_tree_node *root, uint64_t event_id);
 
 void test_sending_event(uint64_t event_id, uint16_t dest_id);
+void sending_event(uint64_t event_id, uint16_t dest_id, struct rte_mbuf *pkt);
+void sending_event_data(uint64_t event_id, uint16_t dest_id, void *pkt);
 int add_event_node_child(struct event_tree_node *parent, struct event_tree_node *child);
 void publish_new_event(uint64_t event_id);
+void publish_event(uint16_t dest_controller,uint64_t event_id);
+void publish_event1(struct event_tree_node** events, struct event_tree_node *root, uint64_t event_id);
```

Comment:
I'm not sure what the differences between these are. Are they just for testing or do we need both of them?